ghe-app-manifest.template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ghe-app-for-jira",
```

Comment:
We probably want this to be an empty string right? Otherwise this will auto-fill the input for every customer which they will then have to delete.

Reply:
Don't we want to give any relevant default name?

Reply:
it's not a bad idea, however, something to consider is that the name has to be unique per server...

Reply:
Caught up with Jade on this one and she agrees it should be blank. From a UX standpoint, we'd only pre-populate an input field if it wasn't clear what a user is meant to enter. However, is this case, app name is pretty straightforward. Plus, to @mboudreau's point: GH app names need to be unique per server so they second a customer tries to create a second app, this would fail. 

Just out of curiosity, what happens when you try and add an existing app name in the input? Guessing GH renders some kind of error but want to be sure.